scripts/on_sd_start.sh

```diff
@@ -80,7 +94,17 @@ else
     export PYTHONPATH="$INSTALL_ENV_DIR/lib/python3.8/site-packages"
 
     if [ "$OS_NAME" == "linux" ]; then
-        if python -m pip install --upgrade torch==1.13.1+cu116 torchvision==0.14.1+cu116 --extra-index-url https://download.pytorch.org/whl/cu116 ; then
+        # Check for AMD dGPU first, assume nobody has a primary nvidia GPU and AMD secondary GPU,
+        # and if they do they can edit this themselves like AMD users already have to
+        if echo "$DGPU_BRAND" | grep -q "AMD"; then
+            installed=$(python -m pip install --upgrade torch torchvision --extra-index-url "https://download.pytorch.org/whl/rocm5.4.2")
```

Comment:
Suppressing the output of pip is making debugging of user errors more difficult. Collecting the return code instead of stdout should be sufficient.